integration/unplugin/src/index.ts

```diff
@@ -252,6 +256,20 @@ const civetUnplugin = createUnplugin((options: PluginOptions = {}) => {
 
         return null;
       },
+      async transformIndexHtml(html) {
+        const cheerio = await import('cheerio');
+        const $ = cheerio.load(html);
```

Comment:
Can we avoid the `cheerio` dependency? Civet aims to have a minimum number of dependencies. I think in this case we could probably write a careful regex...